src/components/Callout/CalloutContent.tsx

```diff
@@ -171,6 +194,45 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
           slideDirectionalClassName: positionInfo.directionalClassName
         });
       }
+    } else {
+      this._positionAttempts = 0;
+    }
+  }
+
+  private _getBounds(): IRectangle {
+    if (!this._bounds) {
+      let currentBounds = this.props.bounds;
+
+      if (!currentBounds) {
+        currentBounds = {
+          top: 0,
+          left: 0,
+          right: this._targetWindow.innerWidth,
+          bottom: this._targetWindow.innerHeight,
+          width: this._targetWindow.innerWidth,
+          height: this._targetWindow.innerHeight
+        };
+      }
+      this._bounds = this._modifyBoundsForPadding(currentBounds);
+    }
+    return this._bounds;
+  }
+
+  private _modifyBoundsForPadding(bounds: IRectangle): IRectangle {
```

Comment:
Yes it can be. I just like lots of methods I guess in this case it's really not necessary.